proctl/proctl.go

```diff
@@ -143,13 +192,19 @@ func (dbp *DebuggedProcess) FindLocation(str string) (uint64, error) {
 		return fn.Entry, nil
 	}
 
-	// Attempt to parse as number for breakpoint id or raw address
+	// No joy
+	return 0, nil
+}
+
+// Find a breakpoint by string (breakpoint id)
+func (dbp *DebuggedProcess) FindBp(str string) (uint64, error) {
+
+	// Attempt to parse number to use as breakpoint id
 	id, err := strconv.ParseUint(str, 0, 64)
 	if err != nil {
-		return 0, fmt.Errorf("unable to find location for %s", str)
+		return 0, fmt.Errorf("unable to find bp %s", str)
```

Comment:
s/bp/breakpoint/

Also, this would actually be clearer as "invalid breakpoint id %s".